framework/src/problems/EigenProblem.C

```diff
@@ -346,6 +354,21 @@ EigenProblem::computeResidualAB(const NumericVector<Number> & soln,
 
   computeResidualTags(_fe_vector_tags);
 
+  if (bxNormProvided())
+  {
+    if (tagA == _nl_eigen->eigenVectorTag())
+      residualA *= (-*_bx_norm);
+    if (tagB == _nl_eigen->eigenVectorTag())
+      residualB *= (-*_bx_norm);
+  }
+  else
+  {
+    if (tagA == _nl_eigen->eigenVectorTag())
+      residualA *= -1;
+    if (tagB == _nl_eigen->eigenVectorTag())
+      residualB *= -1;
+  }
```

Comment:
what is happening here? Why this addition?

Reply:
This is to remove the `-1/k` factor in the residual vector with `eigenVectorTag()` to restore the current behavior.